generator/spec.libsonnet

```diff
@@ -0,0 +1,85 @@
+{
+  // Fix a few issues in the openapi spec so it renders properly, see individual function descriptions for more details.
+  process(spec):
+    self.renameTitleToDescription(spec)
+    + self.addIsPausedToProvisionedAlertRule()
+    + self.removeRecursiveRefOnRoute()
+    + self.fixMatchType(),
+
+  // the OpenAPI schema currently holds a number of `title` fields that contain a description of a parameter, this is used for godoc comments. This lacks standardisation upstream but should dissapear over time. In mean time we'll merge it here with `description` so it shows up in Grafonnet docs.
+  renameTitleToDescription(spec):
+    local schemas = spec.components.schemas;
+    spec + {
+      components+: {
+        schemas+:
+          std.foldl(
+            function(acc, k)
+              acc + {
+                [k]: schemas[k] + {
+                  description:
+                    std.join(
+                      '\n\n',
+                      std.prune([
+                        std.get(schemas[k], 'title', null),
+                        std.get(schemas[k], 'description', null),
+                      ])
+                    ),
+                },
+              },
+            std.objectFields(schemas),
+            {}
+          ),
+      },
+    },
+
+  // Add isPaused to ProvisionedAlertRule, this is missing in v9.4.0
```

Comment:
do these fixups have issues upstream? be good to include references

Reply:
This is for v9.4.0, so not bothering, schema fixes have generally not been backported in grafana/grafana.